health/micro-ui/web/micro-ui-internals/packages/modules/health-dss/src/pages/employee/L2Main.js

```
@@ -0,0 +1,757 @@
+    },
+  });
+  const { data: nationalInfo, isLoadingNAT } = Digit.Hooks.dss.useMDMS(stateCode, "tenant", ["nationalInfo"], {
+    select: (data) => {
+      let nationalInfo = data?.tenant?.nationalInfo || [];
+      let combinedResult = nationalInfo.reduce((acc, curr) => {
+        if (acc[curr.stateCode]) {
+          acc[curr.stateCode].push(curr);
+        } else {
+          acc[curr.stateCode] = [curr];
+        }
+        return { ...acc };
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)